docs/surveys/2020-jupyterlab-survey.md

```diff
@@ -0,0 +1,363 @@
+# JupyterLab
+
+ - Even if you don't use Jupyter, you can still take this survey. Just indicate this fact in the first question and carry on as best as you can.
+ - Thank you - your participation guides Jupyter's roadmap toward your real-life use cases.
+ - So that you know what to expect, it's comprised of 20 questions (#7 is the only big one) spread across these sections:
+    - Usage patterns.
+    - Data
+    - Visualization.
+    - Scale.
+    - Collaboration.
+
+
+### Usage Patterns
+
+1. How frequently do you use Jupyter?
+    - Daily: 4 or more hours per day.
+    - Daily: 3 or less hours per day. 
+    - Weekly.
+    - Monthly.
+    - I no longer use Jupyter.
+    - I have never used Jupyter.
+
+2. How long have you been using Jupyter?
+    - 2+ years.
+    - 1-2 years.
+    - 6-12 months.
+    - Less than 6 months (welcome =]).
+    - I don't use Jupyter.
+
+3. What languages do you use in Jupyter? (pick up to 4)
+    - C (and derivatives)
+    - Groovy
+    - Java
+    - JavaScript
+    - NodeJS
+    - Perl
+    - PHP
+    - Python
+    - R
+    - Ruby
+    - Rust
+    - Scala
+    - Spark SQL
+    - SQL
+    - TypeScript
+    - ❗I wrap/ use bindings for other languages.
+    - ❗My preferred language is not supported in Jupyter.
+    - Other (please specify)
+
+4. What are your primary job roles when you are using Jupyter? (pick up to 2)
+    - Backend engineer.
+    - Business analyst.
+    - Data engineer.
+    - Data scientist.
+    - Database Admin (DBA).
+    - DevOps.
+    - Financial modeler/ analyst.
+    - Front end/ web development.
+    - Infrastructure engineer/ cloud architect.
+    - Scientist/ researcher.
+    - Student.
+    - Sysadmin.
+    - Teacher/ lecturer.
+    - Tutor/ teaching assistant.
+    - Other (please specify)
+
+5. What are your go-to tools for performing data science, scientific computing, 
+and machine learning on your laptop/ desktop (non-cloud) for data science? (pick up to 3)
+    - Spyder.
+    - PyCharm.
+    - nteract.
+    - Atom.
+    - Emacs.
+    - Zeppelin.
+    - iPython.
+    - VS Code.
+    - JupyterLab
+    - RStudio.
+    - Jupyter Notebook (classic)
+    - Vim.
+    - Sublime Text.
+    - Other (please specify)
+
+6. How do you run and/ or access Jupyter? (pick up to 4)
+    - Run directly on local machine (laptop, desktop).
+    - Through a Python virtual environment (e.g. conda, venv)
+    - Through Docker.
+    - HPC or on-premise server.
+    - Cloud server (e.g. EC2).
+    - JupyterHub.
+    - BinderHub / MyBinder.
+    - Cloud service - AWS: EMR, SageMaker.
+    - Cloud service - Azure: Notebooks, ML Studio.
+    - Cloud service - Databricks
+    - Cloud service - Google: AI Platform, Dataproc.
+    - Cloud service - IBM: Watson.
+    - Google Colab.
+    - Through a 3rd party SaaS app (e.g. Observable, CoCalc)
```

Comment:
I think CoCalc should be listed in the Cloud service section as its own item. I'm not sure why it is listed as a "3rd party", but others are listed individually. I think "SaaS" is probably too technical of an abbreviation to use for this survey - I would suggest either deleting this item (letting them fill out "Other").

ObservableHQ doesn't offer Jupyter access, does it? Is that the Observable mentioned here?

Reply:
Agree CoCalc deserves its own item

Reply:
Will make these adjustments and drop SaaS. I think I had PaaS (clouds) and SaaS, but brought out the PaaS so it makes sense to break out the SaaS too. Will drop Observable.

Reply:
☑️ changes integrated.